.github/workflows/freebsd-13.yml

```diff
@@ -49,3 +49,10 @@ jobs:
       - uses: ./.github/actions/environment
       - name: test
         run: make -f .travis.mk test_freebsd
+      - name: artifacts
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: freebsd-13
+          retention-days: 21
+          path: ${{ env.VARDIR }}/artifacts
```

Comment:
See c820754389eb0764c337b1e39b44ab0074dbb47b.

Reply:
Looks like this limitation is gone. See this [section](https://github.com/ChristopherHX/github-act-runner/tree/58ae37abc6c2244d91822b8ba536aa0a8b829632#known-limitations) in README.md. But likely we need to update our agents on freebsd runners.

Reply:
Agents are updated to `v0.2.0` version where this limitation is not relevant anymore.

Reply:
Artifact creation is tested, works fine.